api/src/migrations/api/1621439031072-AddBboxToProjects.ts

```diff
@@ -0,0 +1,48 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddBboxToProjects1621439031072
+  implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+    -----------------------------------------
+    -- Generates the new column we need
+    -----------------------------------------
+    ALTER TABLE projects ADD COLUMN bbox jsonb;
+
+    UPDATE projects SET bbox = jsonb_build_array(ST_XMax(extent), ST_XMin(extent), ST_YMax(extent), ST_YMin(extent));
```

Comment:
as we don't enforce `extent` to be `not null`, are we ok with having a bbox such as `[null, null, null, null]`?

Reply:
I see there's logic about this in the trigger function - do we want to apply it here too?

Reply:
nice catch